docs/_static/style.css

```diff
@@ -0,0 +1,5 @@
+@import url("theme.css");
+
+.wy-side-nav-search {
+  background-color: #fcf;
```

Comment:
Hmm maybe just #fff?

Reply:
(Or whatever is inherited.)

Reply:
done

Reply:
the default color is very blue, not good contrast:

![image](https://user-images.githubusercontent.com/1217238/62492903-997a3580-b784-11e9-8825-318f8f8c546b.png)